src/view/conditions/__tests__/valueComparison.test.jsx

```diff
@@ -100,16 +100,16 @@ describe('value comparison condition view', () => {
         it('sets settings from form values', async () => {
           userEvent.type(pageElements.getLeftOperandTextBox(), '%foo%');
 
+          userEvent.type(pageElements.getRightOperandTextBox(), '123');
```

Comment:
This test started to fail once I stubbed the `requestAnimationFrame` function. It is something that I don't fully understand, and I found 3 ways to make the test to pass. Moving these lines is only one of the option. Maybe we can talk and see if the other options are better.

Reply:
reverted this change after adding the async RAF shim.